libexec/ocid

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/python3
 
```

Comment:
is this correct? explicit python3?

Reply:
I am confused as this file do not look like this in the code base.

Reply:
code have changed already